powerauth-push-server/src/main/java/io/getlime/push/service/PushMessageSenderService.java

```diff
@@ -96,31 +96,31 @@ public BasePushMessageSendResult sendPushMessage(final String appId, final Mode
                     registerPhaserForMode(phaser, mode);
 
                     final Platform platform = device.getPlatform();
-                    if (platform == Platform.IOS) {
+                    if (platform == Platform.IOS || platform == Platform.APNS) {
                         if (pushClient.getApnsClient() == null) {
                             logger.error("Push message cannot be sent to APNS because APNS is not configured in push server.");
                             arriveAndDeregisterPhaserForMode(phaser, mode);
                             continue;
                         }
-                        final PushMessageSendResult.PlatformResult platformResult = sendResult.getIos();
-                        pushSendingWorker.sendMessageToIos(pushClient.getApnsClient(), pushMessage.getBody(), pushMessage.getAttributes(), pushMessage.getPriority(), device.getPushToken(), pushClient.getAppCredentials().getIosBundle(), createPushSendingCallback(mode, device, platformResult, pushMessageObject, phaser));
-                    } else if (platform == Platform.ANDROID) {
+                        final PushMessageSendResult.PlatformResult platformResult = sendResult.getApns();
```

Comment:
Here I fail to see the purpose for the platform result.

Shouldn't we rename the sendMessageToIos to sendMessageToApns?